sklearn/kernel_approximation.py

```diff
@@ -781,8 +791,7 @@ def fit(self, X, y=None):
         U, S, V = svd(basis_kernel)
         S = np.maximum(S, 1e-12)
         self.normalization_ = np.dot(U / np.sqrt(S), V)
-        self.components_ = basis
-        self.component_indices_ = inds
+        self.components_ = X[self.component_indices_]
```

Comment:
Why is this written like this? 
1) self.component_indices_ does not look to be set, and 
2) basis still works as the self.components_
```suggestion
        self.components_ = basis
        self.component_indices_ = basis_inds
```